src/components/Modals/BuyItemSlotsModal/BuyItemSlotsModal.tsx

```diff
@@ -16,6 +16,12 @@ export default class BuyItemSlotsModal extends React.PureComponent<Props, State>
     selectedTierId: undefined
   }
 
+  onCloseModal = (): void => {
```

Comment:
To follow conventions, I think this should be called `handleOnCloseModal` right?

Reply:
Changed it!